google-cloud-notification/pom.xml

```diff
@@ -0,0 +1,198 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>google-cloud-notification</artifactId>
+  <version>0.26.1-alpha-SNAPSHOT</version>
+  <packaging>jar</packaging>
+  <name>Google Cloud Notification</name>
+  <url>https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-notification
+  </url>
+  <description>
+    Java idiomatic client for Google Cloud Notification.
```

Comment:
There is to API called Google Cloud Notification, so we should reword this.

Reply:
Renamed to the header of https://cloud.google.com/storage/docs/pubsub-notifications